### PR TITLE
Idr fix

### PR DESCRIPTION
**Pull request by Maarten-vd-Sande**
**What problem is the PR solving / What's new?**
IDR narrowpeaks did not work with combine_peaks. Now does (by hacking the summit in the middle of the peak). I had some issues with the macs2 and khmer, and adding setuptools fixed it. Not sure why or what was happening there..

**Checklist**
- [x] I made a PR to develop (not master)
- [x] If applicable: I updated the docs
- [ ] I updated the CHANGELOG
- [ ] These changes are covered by the tests

### CODE REVIEW

File: seq2science/envs/macs2.yaml [hunk 1]
```diff
@@ -7,3 +7,4 @@ dependencies:
   - bioconda::macs2=2.2.7
   - bioconda::khmer=2.0
   - conda-forge::jq=1.6
+  - anaconda::setuptools=v50.3.1
```
Comment (siebrenf) [inline seq2science/envs/macs2.yaml:10]: wait wut?

File: seq2science/rules/peak_calling.smk [hunk 2]
```diff
@@ -283,11 +283,15 @@ if "condition" in samples:
             """
             Combine replicates based on the irreproducible discovery rate (IDR). Can only handle two replicates.
             For more than two replicates use fisher's method.
+
+            When combining narrowpeak files we make the q-score zero, and the peak summit in the middle. This is
```
Comment (siebrenf) [inline seq2science/rules/peak_calling.smk:287]: typo and a bit vague. Did you mean "...take the q-score, and the center of the peak summit."

**Review by siebrenf — APPROVED** [review]

File: seq2science/envs/macs2.yaml (hunk 1 above)
Comment (Maarten-vd-Sande) [reply seq2science/envs/macs2.yaml:10]: Yeah I don't know. I got some problems on my own computer with the environment. I googled and the problem was supposed to be caused by an old setuptools. I don't see what caused the problem, but my idea was that this doesn't hurt.

File: seq2science/rules/peak_calling.smk (hunk 2 above)
Comment (Maarten-vd-Sande) [reply seq2science/rules/peak_calling.smk:287]: Nope. IDR makes the qscore and summit both -1, that destroys some downstream tools. So I decided to put the summit in the middle, and the q-score as zero. I'll update the text to be more clear

**Pull request merged**